framework/wazuh/core/config/models/management_api.py

```diff
@@ -91,8 +91,6 @@ class ManagementAPIConfig(WazuhConfigBaseModel):
         Configuration for management API intervals. Default is an instance of ManagementAPIIntervals.
     ssl : APISSLConfig
         SSL configuration for the management API. Default is an instance of APISSLConfig.
-    logging : RotatedLoggingConfig
-        Logging configuration for the management API. Default is an instance of LoggingWithRotationConfig.
```

Comment:
The field still exists but its docstring was removed, I think we should restore it?